### PR TITLE
Fix/accordion restyle

### LINKED ISSUE #421: Restyle vuestic-accordion component
- [x] Restyle VuesticAccordion component according to design
### Design
- Zeplin https://app.zeplin.io/project/5bcd9657767a220982a4ab61/screen/5bd8628fb0f7ad0a51bef74b
- [Design doc](https://xx13.github.io/vuestic_redesign/#accordeon)

### Referencies
- [Vuetify](https://next.vuetifyjs.com/en/components/expansion-panels)
- [Quasar](https://quasar-framework.org/components/collapsible.html#Accordion)

### Referencies
Story points: 2

### PR DESCRIPTION
**Pull request by vlad-shusterman**
close #421

### CODE REVIEW

File: src/vuestic-theme/vuestic-components/vuestic-collapse/VuesticCollapse.vue [hunk 1]
```diff
@@ -11,16 +10,28 @@
...
+        <i
+          v-else
+          class="fa fa-angle-down vuestic-collapse__header__button-arrow"
+        />
+      </div>
     </div>
-
     <div
       class="vuestic-collapse__body"
+      :class="{'background': addBackground}"
```
Comment (AlexanderBoriskin) [inline src/vuestic-theme/vuestic-components/vuestic-collapse/VuesticCollapse.vue]: maybe better to use computed class?
and about naming - maybe 'va-collapse__body--with-background' is better option?

**Comment by asvae** [comment]
* Dispose of `collapse-page__content*` styles.
* Button expand is not needed in presentation.
* Apply default padding to content (see design).
* Add more stuff to issue (document interface, add references, add design doc).
* On presentation option with and without background should be present.
* Tweak demo to only cover cases. Minify it.
* See standard requirements from contribution guide. These should be followed strictly.

**Comment by asvae** [comment]
* Top padding is incorrect for body.
* Padding has transition, when it shouldn't have any.

File: src/vuestic-theme/vuestic-components/vuestic-collapse/VaAccordion.vue [hunk 2]
```diff
@@ -50,7 +50,6 @@ export default {
 
 // HACK Accordion for presentation page
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-collapse/VaAccordion.vue]: Huh?

File: src/vuestic-theme/vuestic-components/vuestic-collapse/VaAccordion.vue [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
   </div>
 </template>
 
 <script>
 export default {
-  name: 'vuestic-accordion',
+  name: 'va-accordion',
   props: {
     expand: {
       type: Boolean,
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-collapse/VaAccordion.vue]: You don't need  `lastClickedTab: {},`

File: src/vuestic-theme/vuestic-components/vuestic-collapse/VaAccordion.demo.vue [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      <va-accordion>
+        <va-collapse>
+          <span slot="header"> Expand This Block </span>
+          <div slot="body">
+            <div>
+              Expand first content
+            </div>
+          </div>
+        </va-collapse>
+        <va-collapse isBackgroundExists>
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-collapse/VaAccordion.demo.vue]: withBackground?

File: src/components/ui/collapse/Collapse.vue [hunk 5]
```diff
@@ -149,15 +169,12 @@ export default {
...
 <style lang="scss">
 .collapse-page {
-  &__content {
-    padding: 2rem;
-    &__title {
-      font-size: 1.375rem;
-      font-weight: bold;
-    }
-    &__description {
+  &__title {
```
Comment (asvae) [inline src/components/ui/collapse/Collapse.vue]: Why not use `.display`?

File: src/vuestic-theme/vuestic-components/vuestic-collapse/VaCollapse.vue [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+        'vuestic-collapse__header--no-header': noHeader,
+      }"
+      @click="onHeaderClick()"
+    >
+      <template v-if="noHeader">
+        <slot name="header"/>
+      </template>
+      <div
+        v-else
+        class="va-collapse__header__content va-row "
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-collapse/VaCollapse.vue]: >  va-row
Huh?

File: src/vuestic-theme/vuestic-components/vuestic-collapse/VaCollapse.vue [hunk 7]
```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="va-collapse">
+    <div
+      class="va-collapse__header"
+      :class="{
+        'vuestic-collapse__header--no-header': noHeader,
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-collapse/VaCollapse.vue]: Not in demo nor presentation.

File: src/vuestic-theme/vuestic-components/vuestic-collapse/VaCollapse.vue [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+        />
+        <i
+          v-else
+          class="fa fa-angle-down va-collapse__header__icon"
+        />
+      </div>
+    </div>
+    <div
+      class="va-collapse__body"
+      :class="{'va-collapse__body--with-background': isBackgroundExists}"
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-collapse/VaCollapse.vue]: Use `.va-collapse--with-background > .va-collapse__body` instead. Also keep margins the same for both modes, just change background color.

File: src/vuestic-theme/vuestic-components/vuestic-collapse/VaCollapse.vue [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+        this.expand()
+      } else {
+        this.collapse()
+      }
+    },
+  },
+  methods: {
+    expand () {
+      const bodyContent = this.$refs.collapseBody
+      bodyContent.style.height = this.$slots.body[0].elm.clientHeight + 36 + 'px'
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-collapse/VaCollapse.vue]: Is that really needed?

Comment heavily if yes.

File: src/vuestic-theme/vuestic-sass/resources/_variables.scss [hunk 10]
```diff
@@ -35,6 +35,7 @@ $dark-gray: #282828;
 $gray: #adb3b9;
 $dark-blue: #0045b6;
 $text-gray: #b4b4b4;
+$collapse-background: #f5f8f9;
```
Comment (asvae) [inline src/vuestic-theme/vuestic-sass/resources/_variables.scss]: Use existing color. (always search before adding color)

File: src/vuestic-theme/vuestic-components/vuestic-collapse/VaCollapse.vue (hunk 9 above)
Comment (vlad-shusterman) [reply src/vuestic-theme/vuestic-components/vuestic-collapse/VaCollapse.vue]: Yes, that's really needed.

File: src/components/ui/collapse/Collapse.vue [hunk 11]
```diff
@@ -1,163 +1,95 @@
 <template>
   <div class="collapse-page">
```
Comment (asvae) [inline src/components/ui/collapse/Collapse.vue]: Docs not linked.

File: src/components/ui/collapse/Collapse.vue [hunk 12]
```diff
@@ -1,163 +1,95 @@
...
-                    <h4 class="collapse-page__content__title">March 2018</h4>
-                    <div>
-                      They occur in a variety of habitats,
-                      such as grasslands, savannas, woodlands, thorny
-                      scrublands.
-                    </div>
-                  </div>
-                </div>
-              </div>
+    <vuestic-widget :headerText="$t('collapse.basic')">
```
Comment (asvae) [inline src/components/ui/collapse/Collapse.vue]: Doc page name and presentation page name differ.

Comment (asvae) [reply src/components/ui/collapse/Collapse.vue]: From docs: 

> value - Boolean (default: false) - controls the opened/closed state of content

This is a incorrect.

File: src/components/ui/collapse/Collapse.vue [hunk 13]
```diff
@@ -1,163 +1,95 @@
...
-                    </div>
-                  </div>
-                </div>
-              </div>
+          </div>
+        </va-collapse>
+      </va-accordion>
+    </vuestic-widget>
+    <vuestic-widget :headerText="$t('collapse.collapseWithBackground')">
+      <va-accordion expand>
```
Comment (asvae) [inline src/components/ui/collapse/Collapse.vue]: Showcase this.

**Pull request merged**